modules/programs/pet.nix

```diff
@@ -0,0 +1,146 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.pet;
+
+  snippetType = types.submodule {
+    options = {
+      description = mkOption {
+        type = types.str;
+        default = "";
+        example = "Show expiration date of SSL certificate";
+        description = ''
+          description of the snippet
+        '';
+      };
+
+      command = mkOption {
+        type = types.str;
+        default = "";
+        example =
+          "echo | openssl s_client -connect example.com:443 2>/dev/null |openssl x509 -dates -noout";
```

Comment:
Could we have a shorter example? I'm a bit worried that this will look a bit bad in the `configuration.nix` man page.